test/test_verifyLeaf.js

```diff
@@ -0,0 +1,65 @@
+const { expect } = require("chai")
+const { ethers } = require("hardhat")
+const { MerkleTree } = require("merkletreejs")
+const generateSampleData = require("../src/sampleData")
+const { buf2hex, getMerkleRoot, keccak, hex2buf } = require("../src/utils")
+
+
+describe("Verification contract", function() {
+    let verification
+    // let hashedData
+    let hexData
+
+    let leafValueBytes = [0, 128, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 0, 36, 6, 0, 0, 0, 0, 0, 0, 0, 0];
+    let leafHash = "0xc09d4a008a0f1ef37860bef33ec3088ccd94268c0bfba7ff1b3c2a1075b0eb92";
+    let proof = {
+        leafIndex: 5,
+        leafCount: 7,
+        items: [
+            "0xe53ee36ba6c068b1a6cfef7862fed5005df55615e1c9fa6eeefe08329ac4b94b",
+            "0x99af07747700389aba6e6cb0ee5d553fa1241688d9f96e48987bca1d7f275cbe",
+            "0xaf3327deed0515c8d1902c9b5cd375942d42f388f3bfe3d1cd6e1b86f9cc456c",
+        ]
+    }
+
+     // This occurs off-chain in the test
+
+     // 1. should_verify (test): https://github.com/paritytech/substrate/blob/td-mmr/frame/merkle-mountain-range/src/tests.rs#L229
```

Comment:
Might be worth linking to a specific commit hash, the branch is no longer in the repo.

Reply:
Updated in https://github.com/Snowfork/verification-solidity/pull/8/commits/0f3a61df73c650a7efe2e496f99b77da88b85266

Reply:
Actually, `test_verifyLeaf.js` is a legacy test file that shouldn't be in this PR. I removed it in https://github.com/Snowfork/verification-solidity/pull/8/commits/34ae205569f21af681a4257cbe87ffc27791d60d. The relevant test file for this PR's functionality is `test_mmr_verification.js`.